Source/Charts/Charts/PieChartView.swift

```diff
@@ -397,11 +397,10 @@ open class PieChartView: PieRadarChartViewBase
                 paragraphStyle.alignment = .center
                 
                 attrString = NSMutableAttributedString(string: newValue!)
-                attrString?.setAttributes([
-                    NSAttributedStringKey.foregroundColor: NSUIColor.black,
-                    NSAttributedStringKey.font: NSUIFont.systemFont(ofSize: 12.0),
-                    NSAttributedStringKey.paragraphStyle: paragraphStyle
-                    ], range: NSMakeRange(0, attrString!.length))
+                attrString?.setAttributes([.foregroundColor: NSUIColor.black,
```

Comment:
shall we use another line?

Reply:
I looked through the framework, and it was very inconsistent for this case. I decided to go with the one I thought was cleanest.

Reply:
To follow the code style of the `{` it should look like this:
```
                attrString?.setAttributes([
                    .foregroundColor: NSUIColor.black,
                    .font: NSUIFont.systemFont(ofSize: 12.0),
                    .paragraphStyle: paragraphStyle
                    ],
                                          range: NSMakeRange(0, attrString!.length))
```

Reply:
sure. NP